app/src/lib/stats/stats-store.ts

```diff
@@ -1670,6 +1671,7 @@ export class StatsStore implements IStatsStore {
       case MultiCommitOperationKind.Reorder:
         return this.recordReorderUndone()
       case MultiCommitOperationKind.CherryPick:
+        return this.recordCherryPickUndone()
```

Comment:
After this I assume `recordCherryPickUndone` can be made `private`?

Reply:
Good catch